cli/src/main/resources/application.conf

```diff
@@ -1,10 +1,33 @@
 msbConfig {
 
+  #ServiceDetails
+  serviceDetails = {
+     name = "msb_java_cli"
+     version = "1.0.1"
+     instanceId = "msbjcli-ed59-4a39-9f95-811c5fb6ab87"
+   }
+
+  # Static Defaults
+  //brokerAdapterFactory = "" //Empty value equals to io.github.tcdl.adapters.mock.MockAdapterFactory
+  brokerAdapterFactory = "io.github.tcdl.adapters.amqp.AmqpAdapterFactory"
+
+  # Thread pool used for scheduling ack and response timeout tasks
+  timerThreadPoolSize: 2
+
+  # Enable/disable message validation against json schema
+  validateMessage = true
```

Comment:
This parameter is not used in CLI tool. Let remove it because its presense is misleading